src/qt/optionsdialog.cpp

```diff
@@ -293,12 +293,21 @@ void OptionsDialog::on_resetButton_clicked()
     }
 }
 
+
 void OptionsDialog::on_openBitcoinConfButton_clicked()
 {
     /* explain the purpose of the config file */
-    QMessageBox::information(this, tr("Configuration options"),
-        tr("The configuration file is used to specify advanced user options which override GUI settings. "
-           "Additionally, any command-line options will override this configuration file."));
+    int ret = QMessageBox::information(this, tr("Configuration options"),
+                                       tr("The configuration file is used to specify advanced user options which override GUI settings. "
```

Comment:
While these lines are touched, maybe add a commit with translator comments?

Reply:
Added translator comments in 25bee9251374fa78d19077f2fb5fa0bfc5d651a2